cmd/kubeadm/app/cmd/init.go

```diff
@@ -32,20 +32,21 @@ import (
 )
 
 var (
-	init_done_msgf = dedent.Dedent(`
+	initDoneMsgf = dedent.Dedent(`
 		Kubernetes master initialised successfully!
 
-		You can connect any number of nodes by running:
+		You can now join any number of machines by running the following on each node:
 
 		kubeadm join --token %s %s
 		`)
 )
 
+// NewCmdInit returns "kubeadm init" command.
 func NewCmdInit(out io.Writer, s *kubeadmapi.KubeadmConfig) *cobra.Command {
 	advertiseAddrs := &[]string{}
 	cmd := &cobra.Command{
 		Use:   "init",
-		Short: "Run this on the first server you deploy onto.",
+		Short: "Run this on the first machine.",
```

Comment:
Please can you add a `Long` description here, which explains what will happen when you run this command?